plugin/src/main/resources/data/lang/en.yml

```
@@ -74,4 +79,12 @@ messages:
     teamBasedStocks:
-      winner: "{lang:prefix} <#e8edff>Team <#a6e3a1>{winners}</#a6e3a1> <#e8edff>won <#8bd5ff>{lang:minigames.{minigameId}.name}</#8bd5ff>!"
\ No newline at end of file
+      winner: "{lang:prefix} <#e8edff>Team <#a6e3a1>{winners}</#a6e3a1> <#e8edff>won <#8bd5ff>{lang:minigames.{minigameId}.name}</#8bd5ff>!"
+  passives:
+    hunger:
+      attackToRestore: "<red>Attack other players to restore hunger!</red>"
+  projectiles:
+    arrow: "Arrow"
+    barrage_arrow: "Barrage Arrow"
+server:
+  motd: "The brawl reborn -- a wondrous thing!"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)